video/csputils.h

```diff
@@ -121,7 +121,8 @@ struct mp_colorspace {
     enum mp_csp_levels levels;
     enum mp_csp_prim primaries;
     enum mp_csp_trc gamma;
-    float peak; // 0 = auto/unknown
+    float nom_peak; // nominal (absolute) peak. 0 = auto/unknown
+    float ref_peak; // reference (white) peak
```

Comment:
Probably doesn't matter in practice, but should these fields be double since they are sourced from integer fractions in the next commit?

Also, vf_format should probably eb able to adjust the new field, but that doesn't need to be included this commit.